src/main/java/com/chikli/hudson/plugin/naginator/NaginatorAction.java

```diff
@@ -42,6 +45,16 @@ public NaginatorAction(@CheckForNull Run<?, ?> parentBuild, int retryCount, int
         this.maxRetryCount = maxRetryCount;
     }
 
+    public void setCause(Cause cause) {
+        this.cause = cause;
+    }
+
+    public String getTooltip() {
+        if (cause != null) {
+            return cause.getShortDescription();
+        }
+        return "";
```

Comment:
`"Re-scheduled after failure"` for backward compatibility.

Reply:
done